api_testing/apis/test_plan_api.py

```diff
@@ -320,11 +320,14 @@ def create_testplan_with_article_not_all(self):
         else:
             raise Exception(f'cant create the test plan with payload {payload}')
 
-    def create_completed_testplan_random_data(self):
-        random_article = random.choice(ArticleAPI().get_all_articles_json())
-        formatted_article = {'id': random_article['id'], 'text': random_article['name']}
-        material_type_id = GeneralUtilitiesAPI().get_material_id(random_article['materialType'])
-        formatted_material = {'id': material_type_id, 'text': random_article['materialType']}
+    def create_completed_testplan_random_data(self, formatted_article='', material_type=''):
```

Comment:
Why you updated this function while you can use function of: 
create_completed_testplan at line 228

Reply:
@ghada-37  create_complete_testplan wasn't working before and this function do the same as i want and i added the parameter as i want specific article